schemas/triniti/canvas/mixin/iframe-block/latest.xml

```diff
@@ -10,6 +10,9 @@
       <field name="width" type="string" pattern="^\d+(%|px)?$"/>
       <field name="height" type="string" pattern="^\d+(%|px)?$"/>
       <field name="align" type="string" pattern="^(center|right|left)$"/>
+      <field name="scrolling_enabled" type="boolean">
+        <default>false</default>
```

Comment:
@nanetteCodes - remove the `false` default value since the default is falsy anyway, i.e. `null`